### PR TITLE
Fix `no-curly-component-invocation` to not warn about invocations within an existing angle bracket invocation

### PR DESCRIPTION
**Pull request by lifeart**
https://github.com/ember-template-lint/ember-template-lint/issues/883

### CODE REVIEW

File: test/unit/rules/lint-no-curly-component-invocation-test.js [hunk 1]
```diff
@@ -29,9 +29,18 @@ generateRuleTests({
      {{/each}}`,
     `{{#-in-element}}Hello{{/-in-element}}`,
     `{{#in-element}}Hello{{/in-element}}`,
+    '<MyComponent @arg={{my-helper this.foobar}} />',
+    '<MyComponent @arg="{{my-helper this.foobar}}" />',
+    '<MyComponent {{my-modifier this.foobar}} />',
```
Comment (Panman8201) [inline test/unit/rules/lint-no-curly-component-invocation-test.js]: Should the single quotes be ticks instead? Or perhaps the ticks should be single quotes (looks like there are both). 🤷‍♂

Comment (rwjblue) [reply test/unit/rules/lint-no-curly-component-invocation-test.js]: yeah the consistency here is a bit wonky (not introduced by @lifeart in this PR though), we should do a cleanup pass and see about enforcing consistent quoting via eslint...

Comment (bmish) [reply test/unit/rules/lint-no-curly-component-invocation-test.js]: Fixed: https://github.com/ember-template-lint/ember-template-lint/pull/891

**Comment by rwjblue** [comment]
Awesome thank you!

**Pull request merged**